File: backend/tests/fixtures/parsers/objc/hello.m
#import <Foundation/Foundation.h>

@interface Greeter : NSObject
- (NSString *)greet:(NSString *)name;
- (NSString *)formatName:(NSString *)n;
@end

@implementation Greeter
- (NSString *)greet:(NSString *)name {
    return [self formatName:name];
}
- (NSString *)formatName:(NSString *)n {
    return [n uppercaseString];
}
@end

File: backend/tests/fixtures/parsers/dart/hello.dart
import 'package:flutter/material.dart';

class Greeter extends Base {
  String greet(String name) {
    return formatName(name);
  }

  String formatName(String n) {
    return n.toUpperCase();
  }
}

void standaloneHelper() {
  final g = Greeter();
  g.greet("world");
}
